clar2wasm/src/tools.rs

```diff
@@ -315,6 +316,27 @@ pub fn crosscheck_compare_only(snippet: &str) {
     );
 }
 
+/// Advance the block height to `count`, and uses the same TestEnvironment instance
```

Comment:
I think "*same*" would be confusing and implies we have one environment that we reuse for both runs.
```suggestion
/// Advance the block height to `count`, and uses identical TestEnvironment copies
```

Reply:
Done